aea/cli/launch.py

```diff
@@ -82,40 +79,24 @@ def _launch_subprocesses(click_context: click.Context, agents: List[Path]) -> in
     :param agents: list of paths to agent projects.
     :return: execution status
     """
-    ctx = cast(Context, click_context.obj)
-    processes = []
-    failed = 0
-    for agent_directory in agents:
-        process = Popen(  # nosec
-            [sys.executable, "-m", "aea.cli", "-v", ctx.verbosity, "run"],
-            cwd=str(agent_directory),
-        )
-        logger.info("Agent {} started...".format(agent_directory.name))
-        processes.append(process)
+    launcher = AEALauncher(
+        agents, mode="multiprocess", fail_policy=ExecutorExceptionPolicies.log_only
+    )
 
     try:
```

Comment:
Seems like the two methods can be merged?

Reply:
actually i want those methods to stay separated for a while, cause it uses different approaches:
AEALauncher and AEARuntime and work around for threads on windows.
I'll open an issue to refactor this part.